WebRoot/js/zimbraMail/share/view/dialog/ZmFolderPropsDialog.js

```diff
@@ -267,13 +267,15 @@ function(event) {
     var retentionTabButton = this._tabContainer.getTabButton(tabKey);
     var retentionIndex = this._tabKeyMap[ZmFolderPropsDialog.TABKEY_RETENTION];
 
-    if ((organizer.type != ZmOrganizer.FOLDER) || organizer.link) {
+    if ((organizer.type != ZmOrganizer.FOLDER) || organizer.link || typeof retentionIndex === "undefined") {
         // Not a folder, or shared - hide the retention view (and possibly the toolbar)
         this._tabInUse[retentionIndex] = false;
         if (this._tabViews.length > 2) {
             // More than two tabs, hide the retention tab, leave the toolbar intact
             tabBar.setVisible(true);
-            retentionTabButton.setVisible(false);
+			if(retentionTabButton) {
+				retentionTabButton.setVisible(false);
+			}
```

Comment:
1. Fix indent
2. add a white space
```suggestion
			if (retentionTabButton) {
				retentionTabButton.setVisible(false);
			}
```
3. Don't we need to care lines 284-289?
```
    } else {
        // Using the retention tab view - show the toolbar and all tabs
        this._tabInUse[retentionIndex] = true;
        retentionTabButton.setVisible(true);
        tabBar.setVisible(true);
    }
```

Reply:
Done point 1 & 2. 
regarding point 3 - those lines will not be executed when retention policy is disabled.